packages/lab/stories/dialog/dialog.stories.tsx

```diff
@@ -20,14 +22,17 @@ export default {
   title: "Lab/Dialog",
   component: Dialog,
   args: {
-    title: "Congratulations! You have created a Dialog.",
+    header: "Congratulations! You have created a Dialog.",
     content:
       "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
   },
 } as Meta<typeof Dialog>;
 
-const DialogTemplate: StoryFn<typeof Dialog> = ({
-  title,
+const DialogTemplate: StoryFn<
+  DialogProps & { header: string; preheader: string }
+> = ({
+  header,
+  preheader,
   // @ts-ignore
```

Comment:
If we're going to the effort of typing header and preheader we should do the same for content